lib/elixir/lib/option_parser.ex

```diff
@@ -1,9 +1,15 @@
 defmodule OptionParser do
   @moduledoc """
-  Functions for parsing command line options.
-
-  The main function in this module is `parse/2`, which allows
-  developers to parse a list of arguments into options:
+  Functions for parsing command line arguments.
+
+  When calling a command, it's possible to pass commandline options
+  to modify what that command does. In this documentation those are
+  called "switches", in other situations they may be called "flags"
+  or simply "options". A switch can be given a value, also called an
```

Comment:
I don't think use "arguments" to refer to switch values in these docs though, do we?

Reply:
Looks like we do that here: https://hexdocs.pm/elixir/OptionParser.html#parse/2-types
"Switches parsed by OptionParser may take zero or one arguments."